cosmwasm/packages/wasmi-runtime/src/registration/cert.rs

```diff
@@ -347,6 +341,40 @@ pub fn verify_ra_cert(cert_der: &[u8]) -> SgxResult<Vec<u8>> {
     Ok(report_public_key)
 }
 
+#[cfg(all(feature = "SGX_MODE_HW", feature = "production"))]
+fn verify_quote_status(quote_status: SgxQuoteStatus) -> Result<(), sgx_status_t> {
+    match quote_status {
+        SgxQuoteStatus::OK => Ok(()),
+        SgxQuoteStatus::SwHardeningNeeded => {
+            warn!("Attesting enclave is vulnerable, and should be patched");
+            Ok(())
+        }
+        _ => {
+            error!("Invalid attestation quote status - cannot verify remote node");
```

Comment:
I would add the reason to this log. doing `{:?}` on the `_` (after renaming it) should work.

Reply:
:+1: